projects/dotcms-field-elements/src/components/dot-multi-select/dot-multi-select.e2e.ts

```diff
@@ -1,100 +1,337 @@
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(optionElements).toEqual([]);
+            });
+        });
+
+        describe('required & requiredMessage', () => {
+            it('should render required attribute in label and dot-required css class', async () => {
+                element.setProperty('required', true);
+                await page.waitForChanges();
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(element).toHaveClasses(['dot-required']);
+                expect(await labelElement.getProperty('required')).toBe(true);
+            });
+
+            it('should render required error msg', async () => {
+                element.setProperty('required', true);
+                element.setProperty('requiredMessage', 'test');
+                await page.waitForChanges();
+                const errorElement = await dotTestUtil.getErrorMessage(page);
+                expect(errorElement.innerHTML).toBe('test');
+            });
+
+            it('should not render required error msg', async () => {
+                const errorElement = await dotTestUtil.getErrorMessage(page);
+                expect(errorElement).toBeNull();
+            });
+
+            it('should not break and not render with invalid data', async () => {
+                const wrongValue = [{ a: 1 }];
+                element.setProperty('required', wrongValue);
+                element.setProperty('requiredMessage', wrongValue);
+
+                await page.waitForChanges();
+
+                const errorElement = await dotTestUtil.getErrorMessage(page);
+                expect(errorElement).toBeFalsy();
+            });
+        });
+
+        describe('value', () => {
+            it('should render option as selected', async () => {
+                element.setProperty('options', 'a|1,b|2');
+                element.setProperty('value', '2');
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(await optionElements[1].getProperty('selected')).toBe(true);
+            });
+
+            it('should render options with no option selected (component\'s default behaviour)', async () => {
+                element.setProperty('options', 'a|1,b|2,c|3');
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(await optionElements[0].getProperty('selected')).toBe(false);
+                expect(await optionElements[1].getProperty('selected')).toBe(false);
+                expect(await optionElements[2].getProperty('selected')).toBe(false);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [{ a: 1 }];
+                element.setProperty('options', 'a|1,b|2,c|3');
```

Comment:
What happen if we pass something like `'a/1-b/2-c/3'`?

Reply:
I'm not saying we should validate exactly the format (we have the doc) BUT we should make sure that we don't leave the value or anything in the field on a bad state.